tests/openthread/Makefile

```diff
@@ -0,0 +1,49 @@
+APPLICATION = openthread
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+CFLAGS += -DDEVELHELP -Wall
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+USEPKG += openthread
+USEMODULE += openthread_contrib
+USEMODULE += libmbedcrypto
+USEMODULE += libopenthread
+USEMODULE += libopenthread-cli
+USEMODULE += xtimer
+
+ifneq (,$(filter samr21-xpro,$(BOARD)))
+  DRIVER := at86rf233
+endif
+ifneq (,$(filter iotlab-m3 fox,$(BOARD)))
```

Comment:
OK but this should not be done here, each board should include its own radio driver otherwise each example will need to add the radio driver for each board!
If we look at gnrc_networking example, DRIVER is not added for each board.
Why is it required here? Because we're in Tests repository?